docs/abap/restful_abap.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: page
+title: Das ABAP Restful Application Programming Model (RAP)
+permalink: /abap/restful_abap/
+parent: abap
```

Comment:
parent müsste hier denke ich nun "Moderne ABAP Entwicklung" sein, damit die Datei korrekt untergeordnet in der Navigation auftaucht.